test/issue1505-single-file-package-dynamic-library.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+cd ${CURR_DIR}
+rm -f libsingle-file-test-dynamic-library.so
+rm -f single-file-test-dynamic-library.dll
+
+${DUB} build --single issue1505-single-file-package-dynamic-library.d
+if [ ! -f libsingle-file-test-dynamic-library.so ] && [ ! -f single-file-test-dynamic-library.dll ]; then
+	die $LINENO 'Normal invocation did not produce a dynamic library in the current directory'
+fi
```

Comment:
The test would be more robust if it checks whether an exported function defined by the single-file shared library yields the expected result when called from another single file executable.

Reply:
@PetarKirov 
My gut feeling is, this rather complex logic is not necessary in this case. When dub and dmd returns with return code 0 and dmd produced a dynamic library, it is safe to assume, the file is valid. A test whether dmd is producing valid dynamic libraries I would see in the test suite of dmd.

Reply:
If we had a test helper script (also used for the rest of the tests) that did such checks my case would be easier to make. In the absence of such, I'm fine with leaving it like this.